From 05dc158e3c632cfd5c7fa20aee6e8373191b45fc Mon Sep 17 00:00:00 2001
From: masem <matej.semian@gmail.com>
Date: Wed, 14 Nov 2018 00:48:39 +0100
Subject: [PATCH] Delay argument "-d" requires number

Arguments "-f -d -n" fails with "invalid number -n", because -d delay expects number of seconds.
---
 modules.d/99shutdown/shutdown.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/modules.d/99shutdown/shutdown.sh b/modules.d/99shutdown/shutdown.sh
index f21cc811..e3884a88 100755
--- a/modules.d/99shutdown/shutdown.sh
+++ b/modules.d/99shutdown/shutdown.sh
@@ -112,17 +112,17 @@ getarg 'rd.break=shutdown' && emergency_shell --shutdown shutdown "Break before
 
 case "$ACTION" in
     reboot|poweroff|halt)
-        $ACTION -f -d -n
+        $ACTION -f -n
         warn "$ACTION failed!"
         ;;
     kexec)
         kexec -e
         warn "$ACTION failed!"
-        reboot -f -d -n
+        reboot -f -n
         ;;
     *)
         warn "Shutdown called with argument '$ACTION'. Rebooting!"
-        reboot -f -d -n
+        reboot -f -n
         ;;
 esac
 

